### ISSUE TITLE
Traits in enums not listed like traits in classes.

### ISSUE THREAD

**Issue by mikebronner** [slevomat/coding-standard#1369]
When including traits in enums, I'm seeing the following warning:

```php
<?php

declare(strict_types=1);

namespace App\Definitions;

use App\Traits\ValueBasedOptions;

enum EmploymentBasis: string
{
    use ValueBasedOptions;

    case FullTime = "Full-Time";
    case PartTime = "Part-Time";
}
```
<img width="353" alt="Screen Shot 2022-05-24 at 3 50 52 PM" src="https://user-images.githubusercontent.com/1791050/170144649-41149442-d527-4a70-958d-1fcc2ca68581.png">

```
Expected 1 line before first use statement, found 0.
(SlevomatCodingStandard.Namespaces.UseSpacing.IncorrectLinesCountBeforeFirstUse)phpcs
```

I was expecting use statements in enums to be treated just like use statements in classes. Thanks!

**Comment by kukulich** [comment]
There’s no support for PHP 8.1 currently. You can try unstable branch “php81”.

**Comment by mikebronner** [comment]
Thanks for the work-around, branch `php81` appears to not have the above issue.
